database/seed.js

```diff
@@ -0,0 +1,82 @@
+const faker = require('faker');
+const db = require('./index.js');
+const Promise = require('bluebird');
+
+const seedRewards = () => {
+  const rewards = [];
+  let tiers;
+  for (let i = 1; i < 101; i++) {
+
+    if (i % 10 === 0 && i % 8 !== 0) {
+      tiers = [1, 5, 10, 25, 50, 75, 100, 250, 500, 750];
+    } else if (i % 8 === 0) {
+      tiers = [1, 5, 10, 25, 50];
+    } else {
+      tiers = [1, 5, 10, 25, 50, 75, 100];
+    }
+
+    for (let j = 0; j < tiers.length; j++) {
+      let isLimited = false;
+      let limitCount = null;
+      let estDeliv = faker.date.month() + '' + faker.random.number({min: 2019, max: 2022});
+      let backers = faker.random.number(500);
+
+      if (j === 8) {
+        isLimited = true;
+        limitCount = 30;
+        backers = faker.random.number(30);
+      }
+      if (j === 9) {
+        isLimited = true;
+        limitCount = 10;
+        backers = faker.random.number(10);
+      }
+
```

Comment:
In the above section, I can't really tell what you're doing because I don't know what the significance of all the number literals are. Why 9? Why 10? Why 30? I would suggest either some named constants or else comments next to the lines where a number literal is used, to explain what it's representing. I think I'd tend towards the named constant, because then you could easily adjust your specifications all throughout the file by just changing that one variable.

Reply:
@jordanstholmes There is absolutely no significance for the number literals. They're just a random number I chose so that every 10th or every 8th project will have a different set of reward tiers. The 10 and the 30 limit are also arbitrarily chosen numbers. The actual site would allow the projectCreator to manually set the number and level of reward tiers (and limit on certain reward tiers if applicable) but since I'm generating a bunch of mock data I just set an arbitrary trap to semi-randomize the data generation (without having completely random reward tier numbers that range all over the place). Maybe I'll add a comment above that section to clarify this point?

Reply:
Coolio. Me over here trying to figure out the pattern all "A Beautiful Mind" style, haha :D